lib/hex_web/package.ex

```diff
@@ -187,11 +187,11 @@ defmodule HexWeb.Package do
       name_search = "%" <> name_search <> "%"
     end
 
-    desc_search = String.replace(search, ~r"\s+", " & ")
+    desc_search = String.replace(search, ~r"\s+", " | ")
 
       from var in query,
     where: ilike(var.name, ^name_search) or
-           fragment("to_tsvector('english', (?->'description')::text) @@ to_tsquery('english', ?)",
+           fragment("to_tsvector('english', regexp_replace((?->'description')::text, '/', ' ')) @@ to_tsquery('english', ?)",
```

Comment:
I'm not sure about this because it feels fragile to just change characters. Is `/` the only character that will cause or will we have to keep tweaking this in the future? I would rather like to tweak the lexing or keep as-is.